slapr/main.py

```diff
@@ -60,10 +64,14 @@ def main(config: Config) -> None:
 
```

Comment:
Since completely removing :review_started: could be a possible breaking change (no high risk though, it's just slack emojis.

What we can possibly do is to not add "review started" when "commented" is defined.

from:
```
new_emojis = {config.emoji_review_started}
```

into:
```
new_emojis = {}
if not config.emoji_commented:
  new_emojis.add(config.emoji_review_started)
```